src/WebJobs.Extensions.EasyTables/Config/EasyTableConfiguration.cs

```diff
@@ -0,0 +1,91 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Configuration;
+using Microsoft.Azure.WebJobs.Host.Config;
+
+namespace Microsoft.Azure.WebJobs.Extensions.EasyTables
+{
+    /// <summary>
+    /// Defines the configuration options for the EasyTable binding.
+    /// </summary>
+    public class EasyTableConfiguration : IExtensionConfigProvider
+    {
+        internal const string AzureWebJobsMobileAppUriName = "AzureWebJobsMobileAppUri";
+        internal const string AzureWebJobsMobileAppApiKeyName = "AzureWebJobsMobileAppApiKey";
+
+
+        /// <summary>
+        /// Constructs a new instance.
+        /// </summary>
+        /// <param name="resolver">The <see cref="INameResolver"/> to use for resolving the MobileAppUri.</param>
+        public EasyTableConfiguration(INameResolver resolver)
+        {
+            this.ApiKey = Resolve(resolver, AzureWebJobsMobileAppApiKeyName);
+
+            string uriString = Resolve(resolver, AzureWebJobsMobileAppUriName);
+
+            // if not found, MobileAppUri must be set explicitly before using the config
+            if (!string.IsNullOrEmpty(uriString))
+            {
+                this.MobileAppUri = new Uri(uriString);
+            }
+
+            this.ClientFactory = new DefaultMobileServiceClientFactory();
+        }
+
+        /// <summary>
+        /// Gets or sets the ApiKey to use with the Mobile App.
+        /// </summary>
+        public string ApiKey { get; set; }
+
+        /// <summary>
+        /// Gets or sets the Mobile App URI for the Easy Table.
+        /// </summary>      
+        public Uri MobileAppUri { get; set; }
+
+        internal IMobileServiceClientFactory ClientFactory { get; set; }
+
+        /// <inheritdoc />
+        public void Initialize(ExtensionConfigContext context)
+        {
+            if (context == null)
+            {
+                throw new ArgumentNullException("context");
+            }
+
+            context.Config.RegisterBindingExtension(
+                new EasyTableAttributeBindingProvider(context.Config, this, context.Config.NameResolver));
+        }
+
+        internal static string Resolve(INameResolver resolver, string key)
+        {
+            string value = null;
+
+            if (resolver != null)
```

Comment:
I think you should remove all this INameResolver stuff for now. I'm not sure we'll change the default resolver to do this - something we'd have to discuss (it'd be a breaking change). So for now remove any temp code that is waiting for that to happen.